src/java.base/share/classes/java/net/SocketOptions.java

```diff
@@ -190,6 +190,13 @@ public interface SocketOptions {
 
     /**
      * See {@link StandardSocketOptions#SO_LINGER} for description of this socket option.
+     * <p>
+     * Set the value to {@code Boolean.FALSE} or an integer less than {@code 0} with
+     * {@link #setOption(int, Object)} to disable this option. An integer greater than or equal to
+     * {@code 0} will enable the option and will represent the linger timeout.
```

Comment:
In the referenced description is it "linger interval" rather than "linger timeout" so I think we should use that here too. Otherwise looks okay.

Reply:
That was an oversight. Fixed it now.